development-environment-base.json

```diff
@@ -8,7 +8,7 @@
     "iso_file": "http://releases.ubuntu.com/16.04/ubuntu-16.04.5-server-amd64.iso",
     "preseed_file": "base.seed",
     "sudo_password": "vagrant",
-    "vagrant_box_file": "build/{{isotime \"2006-01-02-15-10\"}}/{{env `ATLAS_NAME`}}.box",
+    "vagrant_box_file": "build/devenv.box",
```

Comment:
Could this be a (part of a) use-case for someone outside of FinKit?

Reply:
Could be. If it's a feature that's wanted, they'll add it in.